pkg/qsfsd/qsfs.go

```diff
@@ -57,10 +52,12 @@ func New(ctx context.Context, cl zbus.Client, root string) (pkg.QSFSD, error) {
 		return nil, errors.Wrap(err, "couldn't make qsfs mounts dir")
 	}
 
-	return &QSFS{
+	qsfs := &QSFS{
 		cl:         cl,
 		mountsPath: mountPath,
-	}, nil
+	}
+	go qsfs.periodicCleanup(context.Background())
```

Comment:
You should use the passed context

Reply:
The passed context is for the cmdline, the qsfsd service is actually started later with a background context, and I don't know how to access it since it's managed by zbus.